### PR TITLE
Add insertReturnId() for databases that support lastInsertId().

### PR DESCRIPTION
**Pull request by paragonie-scott**
@duskwuff does this look sane?

### CODE REVIEW

**Comment by paragonie-scott** [comment]
This will target `v2.5.0`, the other changes today can either be released earlier as `v2.4.1` or we can just do `v2.5.0` all at once.

File: src/EasyDB.php [hunk 1]
```diff
@@ -427,6 +427,26 @@ public function insert(string $table, array $map): int
...
+     * @return string
+     * @throws Issues\QueryError
+     */
+    public function insertReturnId(string $table, array $map, string $sequenceName = '')
+    {
+        if (!$this->insert($table, $map)) {
+            throw new Issues\QueryError('Could not insert a new row into ' . $table . '.');
+        }
+        if ($sequenceName) {
+            return (string) $this->lastInsertId($sequenceName);
```
Comment (shadowhand) [inline src/EasyDB.php]: This won't work for most situations in Postgres. The way that `PDO::lastInsertId()` works under the hood makes it extremely unreliable and most Postgres users recommend avoiding it entirely.

Comment (paragonie-scott) [reply src/EasyDB.php]: Yeah, should I make it throw an exception if the current driver is pgsql?

**Comment by paragonie-scott** [comment]
This new API is disabled for PostgreSQL. We can disable it for other drivers if need be, down the line. I'm going to tag/sign 2.5.0 today.

**Pull request merged**